modules/references/client/api/references.api.js

```diff
@@ -0,0 +1,70 @@
+import axios from 'axios';
+import set from 'lodash/set';
+import get from 'lodash/get';
+import has from 'lodash/has';
```

Comment:
Flagging for future work to make  `import { set, get, has } from 'lodash'` not to explode bundle size — as I expect this is a workaround for that?

Reply:
I found it in a text about bundle size. But I used it, because importing lodash directly was breaking tests including angular.

Reply:
Oh, funky!

We'll need to look into this more eventually.